terraform/paas/terraform_qa.tfvars

```diff
@@ -1,6 +1,6 @@
 app = {
   name               = "qa-teacher-training-api"
-  docker_image       = "dfedigital/teacher-training-api:testr"
+  docker_image       = "karthikdfe/teacher-training-api-paas"
```

Comment:
did we lose access to the dfe repo?

Reply:
We have now created a specific user `finddockerhub` to push to the right repo.